src/buttons/ExampleButton.ts

```diff
@@ -0,0 +1,14 @@
+import { ButtonHandler } from "../ButtonHandler";
+import { ButtonInteraction } from "discord.js";
+
+class ExampleButton extends ButtonHandler {
+	constructor() {
+		super("exampleButton.myButton");
```

Comment:
I don't like having the whole id for a button. It prevents us from storing data in the button (such as id's, yes or no's). Specifically the polls would be a pain like this.